scripts/cat/cats.py

```diff
@@ -1855,8 +1854,8 @@ def get_ill(self, name, event_triggered=False, lethal=True, severity="default"):
             return
 
         illness = ILLNESSES[name]
-        mortality = illness["mortality"][self.age]
-        med_mortality = illness["medicine_mortality"][self.age]
+        mortality = illness["mortality"][self.age.value]
```

Comment:
Are these necessary? /genq

Reply:
`ILLNESSES` is loaded in from json, so the key for the mortality chance for each age would be a string.